crates/wascc-host/src/messagebus/rpc_client.rs

```diff
@@ -151,19 +190,54 @@ impl Handler<LinkInbound> for RpcClient {
         }
     }
 }
-// Publish a link to the RPC bus
+// Publish a link definition to the RPC bus
 impl Handler<AdvertiseBinding> for RpcClient {
```

Comment:
Should this be renamed to AdvertiseLink, or is that work already being done in https://github.com/wascc/wasccd/pull/12?

Reply:
It should be renamed but you're right, there's already an open PR for that.